jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/transport/HttpSender.java

```diff
@@ -545,6 +545,10 @@ protected Action process() throws Throwable
         @Override
         public void succeeded()
         {
+            // TODO This logic should be moved to process() and/or onCompleteSuccess()
```

Comment:
If this is not planned for this PR maybe this should have a Github issue associated with it and referenced in this comment so it is not forgotten.

Reply:
Turns out that @lorban already has a PR to fix this, so I'll just remove the comment